web/resources/css/main.css

```diff
@@ -27,7 +27,7 @@ body {
 #TOC {
     position: fixed;
     left: 0;
-    top: 0;
+    top: 61px;
```

Comment:
Hm. It's seems, 0 offset is correct. Why 61?

Reply:
offset =  header height